### DISCUSSION TITLE
pconnect connection pool an officially supported feature?

### DISCUSSION THREAD

**Discussion by pysany** [phpredis/phpredis#2236] [Q&A] [answered]
hello. I have a question. Please help me.

From the readme file at https://github.com/phpredis/phpredis
Presistent connections are guided in 'experimental setting'.

However, among the changes in the changelog, it was announced as 'Enable connection pooling by default 8206b147 (Pavlo Yatsukhnenko)' in the [5.0.0] - 2019-07-02 (GitHub, PECL) log.

I trying to use the php connection pool, but it's set to 'experimental setting' so it's not convincing when reporting to my boss.

When I test it, it works without any problem... I wonder if the connection pool function is officially supported.

thank you

![image](https://user-images.githubusercontent.com/41118937/199535791-7f785ff9-bda5-40f8-bd0f-aca09aa137dc.png)

![image](https://user-images.githubusercontent.com/41118937/199535904-e9662825-3562-4f32-adc5-e8e99b0434f1.png)

**Comment by michael-grunder** [comment, marked as answer]
Hi,

There are a couple of different concepts here.  PhpRedis had persistent connections many years before it had logic to pool those connections based on the endpoint.

Either way, I wouldn't call either feature experimental at this point.  It looks like the "experimental setting" line was put into README.md in 2011, so we should probably remove it. :smile:

**Reply by pysany** [reply]
I hope it will be reflected quickly.
thank you for your reply!  😄 😄 😄 😄 😄

**Reply by michael-grunder** [reply]
I've removed the bit about it being experimental 😅
